test/test_thorough_bids_example.py

```diff
@@ -0,0 +1,96 @@
+import bids2openminds.converter
+import os
+from openminds import Collection
+import pytest
+
+
+@pytest.fixture
+def test_standard_collection():
```

Comment:
it is a bit confusing to have "test" in the name of a fixture. I suggest calling this function something like `reference_collection`